pkg/runtime/runtime_test.go

```diff
@@ -5651,7 +5651,7 @@ func TestGracefulShutdownActors(t *testing.T) {
 			Version: "v1",
 			Metadata: []componentsV1alpha1.MetadataItem{
 				{
-					Name: "actorStateStore",
+					Name: actorStateStore,
```

Comment:
have this as string.ToUpper? just to test the case insensitivity?

Reply:
Done